governance-exercises/Executive Vote 02.md

```diff
@@ -0,0 +1,162 @@
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The Enhanced Dai Savings Rate (EDSR) will be activated, increasing the DSR to 8%.
+- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 200 million DAI. 
+- The Harbor Trade Credit vault's Debt Ceiling will be set to zero DAI, and the vault will be put into soft liquidation. 
+- AVC Members' Participation Rewards for Q2 will be distributed. 
+- A DAO Resolution pertaining to the Monetalis Clydesdale James Asset Trust 1 (JAT1) structure will be approved. 
+- A Spark Protocol Proxy Spell will be triggered to implement changes to the DAI interest rate strategy, WETH market parameters and DAI market parameters.
+- 2 million DAI of Launch Project funds will be distributed to a wallet controlled by the Accessibility Scope Facilitator for growth and marketing expenses.
+- The New Silver vault's RWA Agreement parameter will be updated. 
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+For voters' convenience, each proposed item is prefaced with links showing its provenance. "Authorization" links to the official document(s) authorizing the proposed item's inclusion in the Executive Vote; this can be a governance poll, the Atlas and/or a Scope Alignment Artifact. "Reasoning" links to a MakerDAO Forum post providing the rationale and additional context for each item. Where needed, "Authorizing Parties" may also be listed; these may include the responsible Scope Facilitator and/or Scope Advisory Council members.
+
+### Enhanced Dai Savings Rate (EDSR) Activation
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmcTRPLx)\
+*Reasoning*: Rune's [Forum post](https://forum.makerdao.com/t/request-for-gov12-1-2-edit-to-the-stability-scope-to-quickly-implement-enhanced-dsr/21405)
+
+The [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope) was recently amended to add a temporary [Enhanced Dai Savings Rate (EDSR)](https://mips.makerdao.com/mips/details/MIP104#3-2-2-enhanced-dai-savings-rate-edsr-) system aimed at boosting Dai demand. The EDSR is determined based on the [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) and DSR utilization rate. If this executive proposal passes, the following action will occur:
```

Comment:
Arguably the Scope link is your authorization, gets a little more complicated where its authorized by a ratified document that can be amended via polls.